guides/SPI.md

```diff
@@ -21,6 +21,21 @@ In order to become a _service provider_, you have two options:
 * Register an application in the Wix Developer Center, and add the SPI component that you wish to implement. 
 * Become a _service provider_ in a single site by implementing the SPI using Velo. 
 
+## Integration with Wix
+In order to enable Wix applications to communicate with your provider application, Wix needs to know some configuration details. You add the configurations via the Dev Center by entering your app’s url and method and other configuration details in the **JSON Editor** in the service’s extension description page. 
+The **Instructions** section of the extension description page provides a list of the required configuration parameters and types. 
+
+![alt text](../media/spi-config.png)
+
+
+In order to integrate the app you registered in the Wix Developer Center, provide configuration details in the **JSON Editor**:
+1. Click on **Components** in the Dashboard sidebar.
+2. Click on the **Add Component** button to drop down a list of component types.
+3. Select the **Integration component** option.
+4. From the **Choose the integration component** window, select the relevant SPI and click the **Add Component** button.
+5. In the **JSON Editor**, configure the configuration parameters by referencing the **Instructions** section. For each configuration parameter, write the  parameter name and value in the JSON editor, for example “appUrl”: “app world.com/getInfo”.
```

Comment:
Some notes here -

> In the **JSON Editor**

This would be one of those uncommon exceptions I mentioned above - here, you'd say _The **JSON Editor** panel_, or not use its proper name and just call it _The JSON editor_ (either is fine here).

> configure the configuration parameters

repetitive

> For each configuration parameter, write the parameter name and value in the JSON editor, for example “appUrl”: “app world.com/getInfo”.

Our audience is professional developers. They don't need instructions on creating a valid JSON key-value pair. \
However, you should give a little more detail on how to use the **Instructions** panel.